components/board/pi/impl/servo.go

```diff
@@ -54,6 +57,16 @@ func init() {
 				if attr.Max > 0 {
 					theServo.max = uint32(attr.Max)
 				}
+				theServo.maxRotation = uint32(attr.MaxRotation)
+				if theServo.maxRotation == 0 {
+					theServo.maxRotation = uint32(servoDefaultMaxRotation)
+				}
+				if theServo.maxRotation < theServo.min {
+					return nil, errors.New("maxRotation is less than minimum")
+				}
+				if theServo.maxRotation < theServo.max {
+					return nil, errors.New("maxRotation is less than maximum")
+				}
```

Comment:
Apologies, I was unclear, I meant the `Validate` function in [components/board/pi/common/common.go](https://github.com/viamrobotics/rdk/pull/1894/files#diff-b7dd69efb85fbdee8b1c61a18437efca6d33e7f6a4c353950e8f0116ed1e385d).

Reply:
It's fine though, it can stay here.